aya/src/maps/mod.rs

```diff
@@ -500,9 +500,11 @@ impl MapData {
                 }
             })?;
 
+        #[allow(trivial_numeric_casts)]
+        let fd = fd as RawFd;
```

Comment:
I assume this will go away with when `bpf_create_map` returns an OwnedFd?

Reply:
Yep, though it might end up relocating elsewhere.